README.md

```diff
@@ -1,6 +1,7 @@
 # MongoDB PHP Driver
 
-[![Build Status](https://api.travis-ci.org/mongodb/mongo-php-driver.png?branch=master)](https://travis-ci.org/mongodb/mongo-php-driver)
+[![Travis Build Status](https://api.travis-ci.org/mongodb/mongo-php-driver.png?branch=master)](https://travis-ci.org/mongodb/mongo-php-driver)
+[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/33rd3thufo89h2yx?svg=true)](https://ci.appveyor.com/project/derickr/mongo-php-driver)
```

Comment:
Does this always default to the master branch?

Reply:
AppVeyor's history is "flat", and AFAIK it just refers to the last build.